src/calculatorLogic.js

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+export function useCalculator() {
+    const [result, setResult] = useState('');
+    const [firstValue, setFirstValue] = useState(0);
+    const [selectedOperator, setSelectedOperator] = useState('');
+    function numKeyClicked(number) {
+        setResult((currentValue) => currentValue += number);
+    };
+    function operatorClicked(operator) {
+        setFirstValue((value) => value = parseFloat(result));
```

Comment:
Why are we changing the value?
```suggestion
        setFirstValue((value) => parseFloat(result));
```

Reply:
sir in this case shouldn't I also omit the value parameter from the function as it is not being used?